src/main/java/pom/NaptolCartPage.java

```diff
@@ -0,0 +1,57 @@
+package pom;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class NaptolCartPage {
+	
+	@FindBy(xpath="//ul[@id='cartData']")private List<WebElement> cartItem;
+	@FindBy(xpath="(//button[@title='Close'])[2]")private WebElement close;
```

Comment:
Close element not used

Reply:
(Close) Element used for Wait for cart items to display. So this element is used.